### PR TITLE
nginx: add German translation

### PR DESCRIPTION
**Pull request by LukBukkit**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Not known

### CODE REVIEW

File: pages.de/common/nginx.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Nginx Webserver.
+> Weitere Informationen: <https://nginx.org/en/>.
+
+- Starte den Server mit der standardmäßigen Konfigurationsdatei:
+
+`nginx`
+
+- Starte den Server mit einer benutzerdefinierten Konfigurationsdatei:
+
+`nginx -c {{konfigurationsdatei}}`
```
Comment (kbdharun) [inline pages.de/common/nginx.md:12]:
```suggestion
`nginx -c {{konfiguration_datei}}`
```
Would this be better?

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. The translations LGTM, I have some suggestions for this page.

File: pages.de/common/nginx.md (hunk 1 above)
Comment (LukBukkit) [reply pages.de/common/nginx.md:12]: Thanks for your suggestion. In German we can combine words by appending them to each other. The word `Konfigurationsdatei` is the established [translation of a config file](https://translate.google.com/?sl=en&tl=de&text=config%20file&op=translate) and is part of the official [Duden](https://www.duden.de/rechtschreibung/Konfigurationsdatei) directory. If you wish, we could change it to `pfad/zu/konfigurationsdatei`.

File: pages.de/common/nginx.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Starte den Server mit einem Präfix für alle relativen Pfaden in der Konfigurationsdatei:
+
+`nginx -c {{konfigurationsdatei}} -p {{präfix/für/relative/pfade}}`
```
[the remark this answers is not in the document]
Comment (LukBukkit) [reply pages.de/common/nginx.md:16]: See my comment above

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**